src/azure-cli/azure/cli/command_modules/security/_help.py

```diff
@@ -514,11 +514,12 @@
 short-summary: Updates the Azure defender plan for the subscription.
 examples:
   - name: Updates the Azure defender plan for the subscription.
+    command: >
+        az security pricing create -n VirtualMachines --tier standard
+
+  - name: Updates the Azure defender plan for the subscription with suplan and extensions
     text: >
-        az security pricing create -n VirtualMachines --tier 'standard'
-  - name: Updates the Azure defender plan for the subscription. (autogenerated)
-    text: az security pricing create -n VirtualMachines --tier 'standard'
-    crafted: true
+        az security pricing create -n VirtualMachines --tier standard --subplan P2 --extensions name=MdeDesignatedSubscription isEnabled=False --extensions name=AgentlessVmScanning isEnabled=True additionalExtensionProperties='{ \\\"ExclusionTags\\\": \\\"[{\\\\\\\"key\\\\\\\":\\\\\\\"A\\\\\\\",\\\\\\\"value\\\\\\\":\\\\\\\"B\\\\\\\"}]\\\" }'
```

Comment:
```suggestion
        az security pricing create -n VirtualMachines --tier standard --subplan P2 --extensions name=MdeDesignatedSubscription isEnabled=False --extensions name=AgentlessVmScanning isEnabled=True additionalExtensionProperties='{\"ExclusionTags\": [{\"key\": \"A\", \"value\": \"B\"}]}'
```
Could you please give it a try? Ref: https://github.com/Azure/azure-cli/blob/dev/doc/quoting-issues-with-powershell.md.

Reply:
Hey,
This will work, but output of the help, here it is the example won't be correct
The goal is to use the help command output so it can be used by users

Reply:
It should work in PowerShell.